lib/step.rb

```diff
@@ -21,6 +21,7 @@ def command_trimmed
              .join(" ")
              .gsub("#{HOMEBREW_LIBRARY}/Taps/", "")
              .gsub("#{HOMEBREW_PREFIX}/", "")
+             .gsub("/usr/bin/", "/")
```

Comment:
Wouldn't that replace `/usr/bin/git` with `/git`?

Reply:
How does this deal with something say in `/Library/Developer/CommandLineTools/usr/bin` (though maybe there's nothing that would do that though that needs worrying about)

Reply:
Yes, yes it would 😁. Fixed!

Reply:
> (though maybe there's nothing that would do that that needs worrying about)

there isn't but handled anyway now